src/main/resources/conf/examples/bundles.xml

```diff
@@ -26,18 +26,22 @@
   <!--
   <registry url="http://localhost:9200" index="registry" auth="/path/to/auth.cfg" />
   -->
-  <registry url="http://localhost:9200" index="registry" />
+  <registry index="registry" auth="/path/to/auth/file">
+    <direct_url>http://localhost:9200</direct_url>
```

Comment:
@al-niessner can we replace with `cognito_client_id` here in the default configs, since that is going to be required in the new implementation (in most cases)? Same goes for the other examples.

Reply:
We can also probably remove this line from the config:

>  NOTE: For AWS Registry, be sure to force port 443, e.g. https://aws-open-search-url:443